ipaclient/install/timeconf.py

```diff
@@ -39,6 +40,28 @@ def __backup_config(path, fstore=None):
         shutil.copy(path, "%s.ipasave" % (path))
 
 
+def get_time_source(options):
+    """
+    While in interactive installation user has to specify NTP server or pool
+    to be used in chrony configuration. This method asks user input on these
+    values in case that they were not specified before installation start.
+    """
+    if ipautil.user_input("Do you want to configure chrony "
+                          "with NTP server or pool address?", False):
+        servers = user_input("NTP source server addresses", allow_empty=True)
+        if servers:  # if user input is not '' (empty)
+            logger.debug("User provided NTP server(s):")
+            options.ntp_servers = servers.split(',')
```

Comment:
Users tend to separate servers by ``, `` (with space after comma). How about ``[server.strip for server in servers.split(',') if server.strip()]`` ?

Reply:
Sure, can do.